src/components/CountDownButtonModal/index.tsx

```
@@ -20,30 +26,32 @@ export default function CountDownButtonModal({
+    setTimeLeft(parseInt(time));
     if (isPaused) togglePause();
   };
 
   // handle timeout
-  let timeoutId;
+  const timeoutId = useRef<NodeJS.Timeout>();
   useEffect(() => {
-    if (open) timeoutId = setTimeout(() => setOpen(false), time * 1000);
-    return () => clearTimeout(timeoutId);
-  }, [open]);
+    if (open) timeoutId.current = setTimeout(() => setOpen(false), parseInt(time) * 1000);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)